src/Forms/FormLabel.js

```diff
@@ -13,7 +24,14 @@ const FormLabel = React.forwardRef(({ required, children, className, disabled, d
 
     const formLabelClasses = classnames(
         'fd-form-label',
-        { 'is-disabled': disabled },
+        {
+            'is-disabled': disabled,
+            'fd-form-label--toggle': isToggle,
+            'fd-form-label--checkbox': isCheckbox,
+            'fd-form-label--radio': isRadio,
```

Comment:
I don't really love this pattern - but checkbox and radio are going to change significantly in a couple more rc rev's so there's no point in really diving into it right now.